desktop/cmp/grid/impl/ColChooserDialog.js

```diff
@@ -14,17 +14,19 @@ export const colChooserDialog = hoistCmp.factory({
     model: uses(ColChooserModel),
     className: 'xh-col-chooser-dialog',
 
-    render({model, className, ...props}) {
+    render({model, className}) {
 
-        if (!model.isOpen) return null;
+        const {isOpen, width} = model;
+        if (!isOpen) return null;
 
         return dialog({
             icon: Icon.gridPanel(),
             title: 'Choose Columns',
             isOpen: true,
             onClose: () => model.close(),
             item: colChooser({model}),
-            className
+            className,
+            style: {width}
```

Comment:
where does height get set?

Reply:
The dialog will flex to the height of the colChooser (but not its width)